app/models/manageiq/providers/network_manager.rb

```diff
@@ -54,9 +54,10 @@ class << model_name
                :class_name  => "ManageIQ::Providers::BaseManager",
                :autosave    => true
 
+    has_many :availability_zones, :primary_key => :parent_ems_id, :foreign_key => :ems_id
```

Comment:
yes,  NetworkManager must have parent_ems, except Nuage, where we are changing the availability_zones call (and others that are delegated)